trento/xml/article_sap_trento.xml

```diff
@@ -2441,20 +2441,17 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
     <para>
       Any &suse; customer, with a registered &sles4sap;&nbsp;15 (SP1 or higher) distribution,
      experiencing an issue with &trentopremium;, can report the problem either directly in the &scc;
-     or through the corresponding vendor, depending on their licensing model. Please report the problem under &sles4sap;&nbsp;15 and component trento.
+     or through the corresponding vendor, depending on their licensing model. Problems must be reported under &sles4sap;&nbsp;15 and component trento.
     </para>
       <para>
-      When opening a support case for Trento, provide the following information, which can be retrieved
-     as explained in section <xref linkend="sec-trento-problemanalysis" />:
+      When opening a support case for Trento, provide the chosen deployment option for &t.server;: &K8s;, systemd or containeraized.</para>
+   <para>In the case of a &K8s; deployment, provide also the output of the Trento support plugin and the scenario dump, as explained in section <xref linkend="sec-trento-problemanalysis" />.
+    </para>
+   <para>In the case of a systemd deployment, provide the status and the journal of the trento-web and trento-wanda services.
+    </para>
+   <para>Finally, in the case of a containerized deployment, provide the logs of the trento-web and trento-wanda containers. Use docker ps to retrieve
+    the IDs of both containers, then docker logs <container_id> to retrieve the corresponding logs.
```

Comment:
```suggestion
   <para>In case of a &K8s; deployment, provide the output of the Trento support plugin and the scenario dump, as explained in section <xref linkend="sec-trento-problemanalysis" />.
    </para>
   <para>In case of a systemd deployment, provide the status and the journal of the trento-web and trento-wanda services.
    </para>
   <para>In case of a containerized deployment, provide the logs of the trento-web and trento-wanda containers. Use <command>docker ps</command> to retrieve
    the IDs of both containers, then <command>docker logs <replaceable>CONTAINER_ID</replaceable> to retrieve the corresponding logs.
```